src/test/java/SalesforceContactTest.java

```diff
@@ -0,0 +1,161 @@
+        System.setProperty("webdriver.chrome.driver", "./src/test/resources/drivers/chromedriver.exe");
+        driver = new ChromeDriver();
+        js = (JavascriptExecutor) driver;
+        vars = new HashMap<String, Object>();
+    }
+
+    @After
+    public void tearDown() {
+        driver.quit();
+    }
+
+    public void login() {
+        driver.get("https://jpcorp-dev-ed.lightning.force.com");
+        driver.manage().window().setSize(new Dimension(968, 680));
+        driver.findElement(By.id("username")).click();
+        driver.findElement(By.id("username")).sendKeys(vars.get("username").toString());
+        driver.findElement(By.id("password")).click();
+        driver.findElement(By.id("password")).sendKeys(vars.get("password").toString());
+        driver.findElement(By.id("Login")).click();
+    }
+
+    @Test
+    public void logout() {
+        {
+            WebElement element = driver.findElement(By.cssSelector(".photoContainer"));
+            Actions builder = new Actions(driver);
+            builder.moveToElement(element).perform();
+        }
+        {
+            WebElement element = driver.findElement(By.tagName("body"));
+            Actions builder = new Actions(driver);
+            builder.moveToElement(element, 0, 0).perform();
+        }
+        driver.findElement(By.cssSelector(".profileTrigger > .uiImage")).click();
+        driver.findElement(By.linkText("Log Out")).click();
+    }
+
+    @Test
+    public void createContact() {
+        vars.put("username", "juampi7237@gmail.com");
+        vars.put("password", "JP123456jp");
+        vars.put("contactName", "New contact");
+        login();
+        driver.findElement(By.cssSelector(".slds-icon-waffle")).click();
+        driver.findElement(By.xpath("//input[@placeholder='Search apps and items...']")).click();
+        driver.findElement(By.xpath("//input[@placeholder='Search apps and items...']")).sendKeys("Contacts");
+        driver.findElement(By.xpath("//a[@data-label='Contacts']")).click();
+        driver.findElement(By.xpath("//div[contains(text(),'New')]")).click();
+        driver.findElement(By.xpath("//input[@name='lastName']")).click();
+        driver.findElement(By.xpath("//input[@name='lastName']")).sendKeys(vars.get("contactName").toString());
+        driver.findElement(By.name("SaveEdit")).click();
+        assertThat(driver.findElement(By.cssSelector(".slds-theme--success")).getText(), is("success\\\\nContact \\\"vars.get(\"contactName\").toString()\\\" was created.\\\\nClose"));
+        logout();
+    }
+
+    @Test
+    public void createContactAll() {
+        vars.put("username", "juampi7237@gmail.com");
+        vars.put("password", "JP123456jp");
+        vars.put("lastName", "lastname");
+        vars.put("firstName", "firstName");
+        login();
+        driver.findElement(By.cssSelector(".slds-icon-waffle")).click();
+        driver.findElement(By.xpath("//input[@placeholder='Search apps and items...']")).click();
```

Comment:
The majority of the locators are xpath, you should look for css selectors to improve on performance.